src/main/java/org/testcontainers/containers/TarantoolContainerClientHelper.java

```diff
@@ -1,88 +1,109 @@
-        return clientBuilder
-                .withCredentials(container.getUsername(), container.getPassword())
-                .withAddress(container.getHost(), container.getPort())
-                .build();
-    }
-
-    /**
-     * Configure or return an already configured client connected to a Cartridge router
-     *
-     * @return a configured client
-     */
-    public TarantoolClient<TarantoolTuple, TarantoolResult<TarantoolTuple>> getClient() {
-        if (!container.isRunning()) {
-            throw new IllegalStateException("Cannot connect to Tarantool instance in a stopped container");
-        }
-        if (clientHolder.get() == null) {
-            clientHolder.compareAndSet(null, createClient());
-        }
-        return clientHolder.get();
-    }
-
-    public CompletableFuture<List<?>> executeScript(String scriptResourcePath) {
+    public Container.ExecResult executeScript(String scriptResourcePath, Boolean sslIsActive, String keyFile, String certFile) throws IOException, InterruptedException {
         if (!container.isRunning()) {
             throw new IllegalStateException("Cannot execute scripts in stopped container");
         }
 
         String scriptName = Paths.get(scriptResourcePath).getFileName().toString();
         String containerPath = normalizePath(Paths.get(TMP_DIR, scriptName));
         container.copyFileToContainer(MountableFile.forClasspathResource(scriptResourcePath), containerPath);
-        return executeCommand(String.format("return dofile('%s')", containerPath));
+        return executeCommand(String.format("return dofile('%s')", containerPath), sslIsActive, keyFile, certFile);
+    }
+
+    public <T> T executeScriptDecoded(String scriptResourcePath, Boolean sslIsActive, String keyFile, String certFile) throws IOException, InterruptedException {
+        Container.ExecResult result = executeScript(scriptResourcePath, sslIsActive, keyFile, certFile);
+
+        if (result.getExitCode() != 0) {
+
+            throw new IllegalStateException(String.format(EXECUTE_SCRIPT_ERROR_TEMPLATE,
+                scriptResourcePath, result.getExitCode(),
+                result.getStderr(), result.getStdout()));
+        }
+
+        return yaml.load(result.getStdout());
     }
 
-    public CompletableFuture<List<?>> executeCommand(String command, Object... arguments) {
+    public Container.ExecResult executeCommand(String command, Boolean sslIsActive, String keyFile, String certFile) throws IOException, InterruptedException {
         if (!container.isRunning()) {
             throw new IllegalStateException("Cannot execute commands in stopped container");
         }
 
-        return getClient().eval(command, Arrays.asList(arguments));
+        command = command.replace("\"", "\\\"");
+
+        String bashCommand;
+        if (keyFile != "" && certFile != "") {
+            bashCommand = String.format("echo \"print(require('yaml').encode(require('net.box').connect({ uri='%s:%d', " +
```

Comment:
This should be a constant., not an inline string. Also, we may use more proper script formatting for easier reading with backslashes or plus signs.

The same for the subsequent lines